logger_core/src/lib.rs

```diff
@@ -48,6 +51,42 @@ pub static INITIATE_ONCE: InitiateOnce = InitiateOnce {
     init_once: OnceCell::new(),
 };
 
+struct LazyRollingFileAppender {
```

Comment:
Please add some doc here (purpose of this struct etc)

Reply:
Done